src/SamplesApp/SamplesApp.Wasm/Properties/launchSettings.json

```diff
@@ -3,28 +3,28 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:55838/",
-      "sslPort": 0
+      "applicationUrl": "http://localhost:54560/",
+      "sslPort": 44326
```

Comment:
Why?

Reply:
Will revert. It was certainly to avoid conflicts with another uno.ui running instance.